src/style.css

```diff
@@ -1,3 +1,69 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  line-height: 60px;
+  background: white;
+}
+
+.nav__list {
+  margin: 0;
+  display: flex;
+  list-style-type: none;
+  padding: 0;
+}
+
+.nav__item {
+  margin-left: 20px;
+  text-decoration: none;
+}
+
+.nav__item:first-child {
+  margin: 0;
+}
+
+.nav__link {
+  display: inline-block;
+  align-items: center;
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: ' ';
+  position: absolute;
+  bottom: 0;
+  display: block;
+  border-radius: 2px;
+  width: 100%;
+  height: 4px;
+  left: 0;
+  background: #00acdc;
+}
+
+img {
+  display: flex;
+  width: 40px;
+  height: 40px;
+}
+
+:hover {
```

Comment:
Avoid using the `:hover` pseudo-class without specifying the selector it should be applied to. It can lead to unexpected behavior if not used correctly. Apply `:hover` to a specific class or element.